iptables/iptables_test.go

```diff
@@ -66,45 +66,46 @@ func contains(list []string, value string) bool {
 	return false
 }
 
-// Create an array of IPTables with different hasWait/hasCheck to
-// test different behaviours
-func mustTestableIptables() []*IPTables {
+// mustTestableIptables returns a list of ip(6)tables handles with various
+// features enabled & disabled, to test compatability.
+// We used to test noWait as well, but that was removed as of iptables v1.6.0
+func mustTestableIptables(t *testing.T) []*IPTables {
 	ipt, err := New()
 	if err != nil {
-		panic(fmt.Sprintf("New failed: %v", err))
+		t.Fatal(fmt.Sprintf("New failed: %v", err))
```

Comment:
If it has a `must` prefix then please panic, otherwise change the prefix. As this is still setting up the test environment, I think it should panic.